lighthouse-core/audits/manifest-short-name.js

```diff
@@ -28,6 +28,9 @@ class ManifestShortName extends Audit {
       category: 'Manifest',
       name: 'manifest-short-name',
       description: 'Manifest contains `short_name`',
+      helpText: 'Add an optional `short_name` to ensure that your app\'s ' +
```

Comment:
Since this audit falls under the PWA aggregation, agreed we should probably remove "optional". The goal is to check of the PWA list.

@kaycebasques I would also slightly modify the description to say that the `short_name` is a requirement for ATHS, and not mention truncation stuff here. We have that in a separate audit.